### PR TITLE
Fix/to string

### PR DESCRIPTION
**Pull request by francescofuggitti**
## Fixes

Fixes some formatting issues with predicates parameters and equality symbols

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: pddl/logic/predicates.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def __hash__(self) -> int:
 
     def __str__(self) -> str:
         """Get the string representation."""
-        return f"({Symbols.EQUAL} {self.left} {self.right})"
+        return f"({Symbols.EQUAL.value} {self.left} {self.right})"
```
Comment (haz) [inline pddl/logic/predicates.py:130]: No chance of getting a test in for this?

File: Pipfile.lock [hunk 2]
```diff
@@ -1141,15 +1036,16 @@
                 "sha256:d00e6be486affb5781468457b21a6cbe848c33ef43f9ea4a73b4882e5f188a44",
                 "sha256:d429c2430c93b7903914e4db9a966c7f2b068dd2ebdd2fa9b9ce094c7d459f33"
             ],
-            "markers": "python_full_version >= '3.7.0'",
+            "markers": "python_version >= '3.7'",
             "version": "==3.0.0"
         },
-        "wcwidth": {
+        "zipp": {
```
Comment (haz) [inline Pipfile.lock:1042]: What's happening with the Pipfile.lock here? I think dependabot will come back at us for using the full version (efficiency something or other), but this package seems new and there are a handful removed (likely from ipython being removed from the Pipfile?)

**Review by haz — CHANGES_REQUESTED** [review]

File: pddl/logic/predicates.py (hunk 1 above)
Comment (haz) [reply pddl/logic/predicates.py:130]: Kinda surprised we didn't have it crop up before :P

File: Pipfile.lock (hunk 2 above)
Comment (francescofuggitti) [reply Pipfile.lock:1042]: sorry, I messed up with the env. I'm going to close this PR and open a fresh one

File: pddl/logic/predicates.py (hunk 1 above)
Comment (francescofuggitti) [reply pddl/logic/predicates.py:130]: Yep! It was a bit surprising also to me.

**Pull request closed without merging**